src/main/java/com/jalasoft/jfc/model/pdf/PdfConverter.java

```diff
@@ -70,30 +70,22 @@
     public FileResponse convert(Param param) throws CommandValueException, ConvertException, ZipJfcException, IOException {
```

Comment:
delete IOException if its possible

Reply:
it is no possible, because metadata uses it